test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Writer/InstanceAnnotationWriterIntegrationTests.cs

```diff
@@ -555,6 +555,40 @@ private void WriteAnnotationAtEndExpandedFeedShouldFail(ODataFormat format)
             testRequestOfSingleton.Throws<ODataException>(Strings.ODataJsonLightWriter_InstanceAnnotationNotSupportedOnExpandedResourceSet);
         }
 
+        [Fact]
+        public void WriteAnnotationAndCountAtStartExpandedFeedShouldPassInJsonLight()
+        {
+            string expectedPayload =
+            "{" +
+                "\"@odata.context\":\"http://www.example.com/$metadata#TestEntitySet/$entity\"," +
+                "\"ID\":1," +
+                "\"ResourceSetNavigationProperty@odata.navigationLink\":\"http://service/navLink\"," +
+                "\"ResourceSetNavigationProperty@odata.count\":10," +
+                "\"ResourceSetNavigationProperty@custom.StartFeedAnnotation\":123" +
+            "}";
+
+            this.WriteAnnotationAndCountAtStartExpandedFeedShouldPass(ODataFormat.Json, expectedPayload, EntitySet);
+        }
+
+        private void WriteAnnotationAndCountAtStartExpandedFeedShouldPass(ODataFormat format, string expectedPayload, IEdmNavigationSource navigationSource)

```

Comment:
>WriteAnnotationAndCountAtStartExpandedFeedShouldPass [](http://example.com/codeflow?start=21&length=52)

it seems this new private method is only used in the new test case. Maybe it's not needed.